apps/group/contracts/Group.sol

```diff
@@ -23,19 +30,32 @@ contract Group is App, Initializable, IForwarder, EVMCallScriptRunner {
     * @notice Initialize new `_name` group
     * @param _name Name for the group
     */
-    function initialize(string _name) onlyInit external {
+    function initialize(string _name, uint _required) onlyInit external {
         initialized();
 
+        required = _required;
         name = _name;
     }
 
+    /**
+    * @notice Changes the number of members required to confirm before the group can perform actions
+    * @param _required New number of members required to confirm action
+    */
+    function changeRequirement(uint _required) public {
```

Comment:
Any group member shouldn't be allowed to change the requirement, it completely defeats the purpose as anyone would be able to set the requirement to 1 and act.

Reply:
Change requirement should have its own role and be protected with the auth modifier